runner/cf-driver-go/cg/cg_integration_test.go

```diff
@@ -0,0 +1,76 @@
+//go:build integration
+
+package cg_integration
+
+import (
+	"bufio"
+	"encoding/json"
+	"os"
+	"testing"
+
+	"github.com/GSA-TTS/gitlab-runner-cloudgov/runner/cf-driver/cg"
+	"github.com/google/go-cmp/cmp"
+)
+
+func Test_CFAdapter_GetApps(t *testing.T) {
+	var u, p, want string
+	var l int
+
+	f, err := os.Open("./testdata/.cg_creds")
+	if err != nil {
+		t.Errorf("Error opening testdata file = %v", err)
+	}
+	defer f.Close()
+
+	scanner := bufio.NewScanner(f)
+
+scanning:
+	for scanner.Scan() {
+		text := scanner.Text()
+
+		if text[0] == '#' {
+			continue
+		}
+
+		switch l {
+		case 0:
+			u = text
+		case 1:
+			p = text
+		case 2:
+			want = text
+			break scanning
```

Comment:
[go question] ah - is there no "break out of the loop" without adding an explicit label?

Reply:
You _can_ but you can also break from a switch so without the label this break wouldn't do anything. I thought this generally looked OK but if it's a bit confusing it might be worth me writing it differently.